budget/tests.py

```diff
@@ -669,18 +670,18 @@ def test_export(self):
                      u'amount': 200.0, u'payer_name': u'fred', u'payer_weight': 1.0, u'owers': [u'alexis', u'tata']},
                     {u'date': u'2016-12-31', u'what': u'fromage \xe0 raclette',
                      u'amount': 10.0, u'payer_name': u'alexis', u'payer_weight': 2.0, u'owers': [u'alexis', u'fred', u'tata', u'p\xe9p\xe9']}]
-        self.assertEqual(json.loads(resp.data), expected)
+        self.assertEqual(json.loads(resp.data.decode('utf-8')), expected)
 
         # generate csv export of bills
         resp = self.app.post("/raclette/edit", data={
             'export_format': 'csv',
             'export_type': 'bills'
         })
-        expected = ["date,what,amount,payer_name,payer_weight,owers",
-                    "2017-01-01,refund,13.33,tata,1.0,fred",
-                    "2016-12-31,red wine,200.0,fred,1.0,\"alexis, tata\"",
-                    "2016-12-31,fromage à raclette,10.0,alexis,2.0,\"alexis, fred, tata, pépé\""]
-        received_lines = resp.data.split("\n")
+        expected = [u"date,what,amount,payer_name,payer_weight,owers",
```

Comment:
I would rather import unicode_literals in the beginning: ``from __future__ import unicode_literals``